### ISSUE TITLE
paper and code differences

### ISSUE THREAD

**Issue by niklas030** [nv-tlabs/planning-centric-metrics#1]
Hey!

I am currently trying to reproduce the planner for the PKL metric. After reviewing both the paper and the repository, I have noticed a few discrepancies in the current code:

- Dataset Size: I am unable to extract the mentioned 1.2M trajectories; I only get around 300k for training, while the validation set appears to be slightly larger (around 600).
- Planner Input: The input logic seems to have changed in the repository. It currently only takes the current timestamp rather than the two seconds of history proposed in the paper.

I understand this is an older repository, but do you happen to have older weights or perhaps another fork or updated branch that would help in reproducing the planner results?

Thank you for your help.

Best regards,

Niklas Schmitt
